thrift/src/main/java/com/linecorp/armeria/server/thrift/THttpService.java

```diff
@@ -689,9 +693,10 @@ private static HttpData encodeSuccess(ServiceRequestContext ctx,
                                           SerializationFormat serializationFormat,
                                           String methodName, int seqId,
                                           TBase<?, ?> result) {
-
-        final TMemoryBuffer buf = new TMemoryBuffer(128);
-        final TProtocol outProto = ThriftProtocolFactories.get(serializationFormat).getProtocol(buf);
+        final ByteBufAllocator alloc = ctx.alloc();
+        ByteBuf buf = alloc.buffer(128);
```

Comment:
`alloc.ioBuffer()` so that a direct buffer is used for potentially fewer memcpys?